src/bus.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+//
+//  events
+//
+
+import EventEmitter from 'events';
+
+export type onMessage = {
```

Comment:
I don't like the name `onMessage`. Probably just call it `Message`, `MessageEvent` or something like that? Or is there some Flow naming convention I don't know about?